services/frontend-service/src/ui/components/chip/EnvironmentGroupChip.tsx

```diff
@@ -49,10 +49,16 @@ export type EnvironmentChipProps = {
     smallEnvChip?: boolean;
 };
 
+const getNamespace = (envCfg: EnvironmentConfig | undefined): string => {
+    const destination = envCfg?.argocd?.destination;
+    return destination?.namespace ?? destination?.applicationNamespace ?? 'tools';
```

Comment:
```suggestion
    return destination?.namespace ?? destination?.applicationNamespace ?? '';
```

```suggestion
    return destination?.namespace ?? destination?.applicationNamespace ?? '';
```
The default in render.go is "", not "tools".

Reply:
The default is now set in the helm chart and defaults to `"tools"` in the frontend service for backwards compatibility.